### PR TITLE
Print using for directive in ASTPrinter

### PR DESCRIPTION
**Pull request by kseo**

### CODE REVIEW

File: libsolidity/ast/ASTPrinter.cpp [hunk 1]
```diff
@@ -78,6 +78,13 @@ bool ASTPrinter::visit(InheritanceSpecifier const& _node)
 	return goDeeper();
 }
 
+bool ASTPrinter::visit(UsingForDirective const& _node)
+{
+	writeLine("UsingDirective");
+	printSourcePart(_node);
```
Comment (axic) [inline libsolidity/ast/ASTPrinter.cpp]: What will this print for a the node? Just `UsingDirective`?

This entire AST printer is not very useful.

File: libsolidity/ast/ASTPrinter.cpp [hunk 2]
```diff
@@ -78,6 +78,13 @@ bool ASTPrinter::visit(InheritanceSpecifier const& _node)
 	return goDeeper();
 }
 
+bool ASTPrinter::visit(UsingForDirective const& _node)
+{
+	writeLine("UsingDirective");
```
Comment (axic) [inline libsolidity/ast/ASTPrinter.cpp]: Please use `UsingForDirective`.

Comment (kseo) [reply libsolidity/ast/ASTPrinter.cpp]: Fixed!

File: libsolidity/ast/ASTPrinter.cpp (hunk 1 above)
Comment (kseo) [reply libsolidity/ast/ASTPrinter.cpp]: I added it for completeness. It will correctly indent the sub nodes of `UsingForDirective`.

**Review by axic — APPROVED** [review]

**Pull request merged**